lib/SymbolGraphGen/SymbolGraph.h

```diff
@@ -216,7 +216,7 @@ struct SymbolGraph {
   ///
   /// \param IgnoreContext If `true`, don't consider
   /// the context of the symbol to determine whether it is implicitly private.
```

Comment:
Are all `Decl`s symbols or just `ValueDecls`? If `Decl`s include more than just symbols (e.g. extensions) then this comment should change.

```suggestion
  /// the context of the declaration to determine whether it is implicitly private.
```